spec/domain/output_document_spec.rb

```diff
@@ -87,7 +87,7 @@
 
       it 'includes the Country' do
         expect(attendee_address).to eq("Mr Joe Bloggs\nHM Treasury\n1 Horse Guards Road\nWhitehall\n" \
-                                       "Westminster\nGreater London\nSW1A 2HQ\n#{country}")
+                                       "Westminster\nGreater London\nSW1A 2HQ\n#{country.upcase}")
```

Comment:
I know you're editing an existing spec but I think this makes more sense to assert with the `end_with` expectation.